packages/react/package.json

```diff
@@ -46,6 +46,7 @@
   "dependencies": {
     "@knocklabs/client": "*",
     "@knocklabs/react-core": "*",
+    "@popperjs/core": "^2.11.8",
```

Comment:
Guessing this is a peer dependency of react-popper?

Reply:
Yeah, it's a peer dependency in react-popper, but we also use it for types in `NotificationFeedPopover.tsx`. Not sure why it being missing didn't break the build. Issue only surfaced when there was a conflict with a user's app which also had react-popper installed.